frontends/web/src/api/accountsync.ts

```diff
@@ -43,3 +45,35 @@ export const syncAddressesCount = (
     cb(code, data);
   });
 };
+
+/**
+ * Fired when status of an account changed, mostly
+ * used as event to call accountApi.getStatus(code).
+ * Returns a method to unsubscribe.
+ */
+export const statusChanged = (
+  code: string,
+  cb: (code: string) => void,
+): TUnsubscribe => {
+  const unsubscribe = subscibeLegacy('statusChanged', data => {
+    if (data.type === 'account' && data.code === code) {
+      cb(code);
+    }
+  });
+  return unsubscribe;
+};
+
+/**
+ * Fired when the account is fully synced.
+ * Returns a method to unsubscribe.
+ */
+export const syncdone = (
+  code: string,
+  cb: (code: string) => void,
+): TUnsubscribe => {
+  return subscibeLegacy('syncdone', data => {
+    if (data.type === 'account' && data.code === code) {
+      cb(code);
+    }
+  });
+};
```

Comment:
statusChanged and syncdone was just moved from api/subscribe-legacy.ts, as it should be in api/accountsync.